app/scripts/modules/appengine/src/serverGroup/configure/wizard/configFileArtifactList.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import { ConfigFileArtifactList } from './ConfigFileArtifactList';
+import { mount } from 'enzyme';
+import { API, IArtifactAccount, IArtifactAccountPair, StageArtifactSelector } from '@spinnaker/core';
+import React from 'react';
```

Comment:
Usually Deck imports get sorted such that 3rd party dependencies (`react`, `enzyme`) go above internal dependencies (`./ConfigFileArtifactList`) with a space between them.

e.g., https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/pipeline/config/stages/deployService/DeployServiceExecutionDetails.tsx#L1-L5

Reply:
I did this manually because all my deck webstorm settings are messed up :/

Reply:
@zachsmith1 you could also run `prettier` yourself on the files. There's the `yarn prettier` command that will format the whole project, or you can borrow the script command and run it yourself on a subset:
`bash -c '$(npm bin)/prettier --write \'app/scripts/modules/kubernetes/src/serverGroupManager/*.{js,jsx,ts,tsx,html,css,less}\''`

☝️ just change the path to whatever and make sure you run the command from the root of the project :)

Reply:
@dogonthehorizon will def give that a try